src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_5001_6000.cpp

```diff
@@ -4147,6 +4147,33 @@ GUI_TEST_CLASS_DEFINITION(test_5790) {
     CHECK_SET_ERR(GTUtilsMcaEditorSequenceArea::getSelectedRect(os) == emptyselection, "Selection isn't empty but should be");
 }
 
+GUI_TEST_CLASS_DEFINITION(test_5794) {
+    // 0. Rename the "data" folder
+    QDir dir(".");
+    auto cp = dir.current();
```

Comment:
nit: using 'auto' in this case (short trivial type names like QDir/QString) only reduce overall readability.

Reply:
This variable is unused, removed